resources/migrations/051_add_new_query_execution_table.yaml

```diff
@@ -0,0 +1,82 @@
+databaseChangeLog:
+  - changeSet:
+      id: 51
+      author: camsaul
+      changes:
+        - createTable:
+            tableName: query_execution
+            remarks: 'A log of executed queries, used for calculating historic execution times, auditing, and other purposes.'
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: hash
```

Comment:
Yeah, I was thinking we could use the Card ID to look it up if we needed it (so far, we've never used it for anything except seeing whether or not it's native in the anon usage stats). If we come up with more advanced uses for it, we can always add the query table like we talked about doing in the past. That would be pretty easy since we can make the query hash the PK